### PR TITLE
chore(react): add `TabPanel` component to `Tabs` component story

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Purpose

> $subject

Updated story:
Screenshot 2023-03-10 at 3 24 20 PM

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] UX/UI review done on the final implementation.
- [ ] Story provided. (Add screenshots)
- [ ] Manual test round performed and verified.
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Documentation provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: packages/react/src/components/Tabs/Tabs.stories.mdx [hunk 1]
```diff
@@ -59,16 +75,34 @@ Import and use the `Tabs` component in your components as follows.
...
-    <Tabs value={value} onChange={handleChange}>
-      <Tab label="Tab One" />
-      <Tab label="Tab Two" />
-      <Tab label="Tab Three" />
-      <Tab label="Tab Four" />
-      <Tab label="Tab Five" />
-    </Tabs>
+    <Box>
+      <Tabs value={value} onChange={handleChange}>
+        <Tab label="Tab One" id="tab-0" aria-controls="tabpanel-0" />
```
Comment (savindi7) [inline packages/react/src/components/Tabs/Tabs.stories.mdx:84]: Any specific reason why there are 5 tabs?

Comment (JayaShakthi97) [reply packages/react/src/components/Tabs/Tabs.stories.mdx:84]: Actually there is no specific reason to add 5 tabs exactly. Anyway we can reduce the number of tabs.

Comment (savindi7) [reply packages/react/src/components/Tabs/Tabs.stories.mdx:84]: Shall we keep it at 3?

Comment (JayaShakthi97) [reply packages/react/src/components/Tabs/Tabs.stories.mdx:76]: Addressed with 3a5b291d9f4ff24274f8cdc7bb988596e2d4c935

**Review by savindi7 — APPROVED** [review]

**Pull request merged**